fiftyone/core/cli.py

```diff
@@ -686,20 +686,46 @@ class DatasetsDeleteCommand(Command):
 
     Examples::
 
-        # Delete the dataset with the given name
-        fiftyone datasets delete <name>
+        # Delete the datasets with the given name(s)
+        fiftyone datasets delete <name1> <name2> ...
+
+        # Delete the datasets whose names match the given glob pattern
+        fiftyone datasets delete --glob-patt <glob-patt>
+
+        # Delete all non-persistent datasets
+        fiftyone datasets delete --non-persistent
     """
 
     @staticmethod
     def setup(parser):
         parser.add_argument(
-            "name", metavar="NAME", help="the name of the dataset",
+            "name",
+            metavar="NAME",
+            nargs="*",
+            help="the dataset name(s) to delete",
+        )
+        parser.add_argument(
+            "-g",
+            "--glob-patt",
+            metavar="GLOB_PATT",
+            help="a glob pattern of datasets to delete",
+        )
+        parser.add_argument(
+            "--non-persistent",
+            action="store_true",
+            help="delete all non-persistent datasetes",
```

Comment:
```suggestion
            help="delete all non-persistent datasets",
```

Reply:
Thanks for catching the typo!

Reply:
`examples/` was deleted because those were moved to https://github.com/voxel51/fiftyone-examples, which isn't public yet. When it is public, we'll add a link from the README here to that repo